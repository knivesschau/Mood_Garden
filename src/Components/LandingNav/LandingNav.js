import React, {Component} from 'react';
import {Link} from 'react-router-dom';
import './LandingNav.css';

export default class LandingNav extends Component {
    render() {
        return (
            <div className="LandingNav">
                <Link to='/register'>
                    Sign Up
                </Link>

                <Link to='/login'>
                    Login
                </Link>

                <Link to='/your-garden'>
                    Your Garden
                </Link>

            </div>
        )
    }
}