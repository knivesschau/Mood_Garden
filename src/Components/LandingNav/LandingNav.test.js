import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingNav from './LandingNav';

describe ('LandingNav Component', () => {
    it ('renders the complete LandingNav Component', () => {
        const wrapper = shallow(<LandingNav/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})


