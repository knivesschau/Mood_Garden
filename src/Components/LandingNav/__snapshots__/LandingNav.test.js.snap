// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingNav Component renders the complete LandingNav Component 1`] = `
<div
  className="LandingNav"
>
  <Link
    to="/register"
  >
    Sign Up
  </Link>
  <Link
    to="/login"
  >
    Login
  </Link>
  <Link
    to="/your-garden"
  >
    Your Garden
  </Link>
</div>
`;
